Probesys/bileto
# new: Allow to assign contracts to tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/27

## Changes



- add new permissions related to tickets' contracts
- add a relation between contracts and tickets
- automatically attach new tickets to ongoing contracts (in TicketsController and in CreateTicketsFromMailboxEmailsHandler)
- display the related ongoing contract in the ticket page
- add a controller to edit a ticket's ongoing contract

## How to test manually



- create an ongoing contract
- create a ticket → check it's attached to the contract
- create a new ongoing contract
- change the contract of the ticket → check it's correctly changed

## Checklist
